source/common-tasks/os.markdown

```diff
@@ -20,4 +20,5 @@ This section will provide guides to some common tasks and information which you
 {% include common-tasks/data_disk.md %}
 {% include common-tasks/flashing_n2_otg.md %}
 {% include common-tasks/enable_i2c.md %}
+{% include common-tasks/enable_entities.md %}
```

Comment:
This is not an OS related specific task...

Reply:
You are right. Moved it to its own page. Let me know what you think.